applications/CompressiblePotentialFlowApplication/tests/cpp_tests/test_operations.cpp

```diff
@@ -0,0 +1,166 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Marco Antonio Zuñiga Perez
+//
+//
+
+// Project includes
+#include "containers/model.h"
+#include "testing/testing.h"
+#include "fluid_dynamics_application_variables.h"
+#include "compressible_potential_flow_application_variables.h"
+#include "custom_operations/potential_to_compressible_navier_stokes_operation.h"
+
+namespace Kratos {
+namespace Testing {
+
+/**
+* @brief This method creates a potential_model_part and a "fake" compressible_model_part. 
+* The potential nodal values are setting and the nodal velocities are passed with 
+* the PotentialToCompressibleNavierStokesOperation to the compressible_model_part.
+* The values are check with the explicit local calculate for each node.
+*/
+KRATOS_TEST_CASE_IN_SUITE(PotentialToCompressibleNavierStokesOperation, CompressiblePotentialApplicationFastSuite)
+{
+    Model this_model;
+    
+    // Create potential_model_part
+    ModelPart& model_part = this_model.CreateModelPart("Main", 3);
+    model_part.GetProcessInfo()[DOMAIN_SIZE] = 2;
+    
+    // Set potential_model_part properties
+    BoundedVector<double, 3> free_stream_velocity = ZeroVector(3);
+    free_stream_velocity(0) = 10.0;      
+    const double& gamma = 1.4;
+    const double& sound_velocity = 340;
+    const double& free_stream_density = 1.0;
+    const double& free_stream_mach = 0.2;
+    const double& reference_temperature = 273;
+    model_part.GetProcessInfo()[FREE_STREAM_VELOCITY] = free_stream_velocity;
+    model_part.GetProcessInfo()[HEAT_CAPACITY_RATIO] = gamma;
+    model_part.GetProcessInfo()[SOUND_VELOCITY] = sound_velocity;
+    model_part.GetProcessInfo()[FREE_STREAM_DENSITY] = free_stream_density;
+    model_part.GetProcessInfo()[FREE_STREAM_MACH] = free_stream_mach;
+    const double& specific_heat = (std::pow(sound_velocity,2) / (gamma * reference_temperature)) / (gamma - 1.0);
+    
+    // Variables addition
+    model_part.AddNodalSolutionStepVariable(VELOCITY_POTENTIAL);
+    model_part.AddNodalSolutionStepVariable(AUXILIARY_VELOCITY_POTENTIAL);
+    
+    // Set the element properties
+    model_part.CreateNewProperties(0);
+    Properties::Pointer pElemProp = model_part.pGetProperties(0);
```

Comment:
```suggestion
    auto p_elem_prop = model_part.CreateNewProperties(0);
```
The create method already returns a pointer to the new properties.